templates/new_password.html

```diff
@@ -11,17 +11,20 @@
 <div class="formular">
 <div class="form-group row">
 
-	<label for="heslo" class="col-sm-2 col-form-label">Heslo*:</label>
+    <label for="password" class="col-sm-2 col-form-label">Heslo*:</label>
 	<div class="col-sm-10">
-	<input type="password" class="form-control form-control-sm" min-width=8 name="heslo" required id="heslo" value="">
+        <input type="password" class="form-control form-control-sm" min-width=8 name="heslo" required id="password"
```

Comment:
You forgot to change the `name` attribute as well - this is what needs to be changed in order for the `result.get()` in `new_password()` function in password_renewal_bp.py to work correctly.

Same for the password confirmation below.